src/blocks.js

```diff
@@ -1852,7 +1852,13 @@
     "block_icon": "./blocks/yashraj2003e/icon.png",
     "author_name": "Yashraj",
     "author_github_url": "https://github.com/yashraj2003e/"
-  }
+  },{
```

Comment:
Please do not alter the `blocks.js` file and follow the guidelines exactly. Change this back to normal please.